vault/diagnose/output.go

```diff
@@ -276,7 +276,7 @@ func (t *TelemetryCollector) getOrBuildResult(id trace.SpanID) *Result {
 	return r
 }
 
-func findAttribute(e trace.Event, attr attribute.Key) string {
+func findAttribute(e sdktrace.Event, attr attribute.Key) string {
```

Comment:
The `Event` type previously lived under `otel/trace`, but has now been moved to `otel/sdk/trace` instead